docs/pages/versions/unversioned/sdk/constants.md

```diff
@@ -44,6 +44,8 @@ Gets the user agent string which would be included in requests sent by a web vie
 
 ### `Constants.installationId`
 
+> ⚠️ **This property is deprecated and will be removed in the SDK 41. Please, implement it on your own using `expo-secure-store` on iOS, `expo-application`'s `androidId` on Android and `localStorage` on Web.**
```

Comment:
Let's give people more time than just one SDK release to rewrite their code. It's not so hard to continue supporting this API for 9+ months.

```suggestion
> ⚠️ **This property is deprecated and will be removed in the SDK 43. Please, implement it on your own using `expo-application`'s `androidId` on Android and a storage API such as `expo-secure-store` on iOS and `localStorage` on Web.**
```

(also would be helpful to hyperlink to the `androidId` section of the docs)